sessionctx/context.go

```diff
@@ -206,3 +208,9 @@ func ValidateStaleReadTS(ctx context.Context, sctx Context, readTS uint64) error
 	}
 	return nil
 }
+
+// SysProcTracker is used to track background sys processes
+type SysProcTracker struct {
```

Comment:
Why not let it be a interface?

Reply:
Yes, you are right, it's better and there's no import cycle either. Done.